builtin/providers/azurerm/resource_arm_virtual_machine.go

```diff
@@ -190,7 +208,24 @@ func resourceArmVirtualMachine() *schema.Resource {
 
 						"vhd_uri": {
 							Type:     schema.TypeString,
-							Required: true,
+							Optional: true,
```

Comment:
Ok I submitted the `ConflictsWith` fix under PR #13109 and removed it from this branch. Do you think it will be alright to leave the `ConflictsWith` setup in place here with the hope that the other PR would get merged in time for the release of this feature or should I revert this back to the custom conflict enforcement logic?